core/arch/arm/kernel/abort.c

```diff
@@ -525,10 +530,12 @@ static enum fault_type get_fault_type(struct abort_info *ai)
 		return FAULT_TYPE_PAGEABLE;
 
 	case CORE_MMU_FAULT_ASYNC_EXTERNAL:
+	case CORE_MMU_FAULT_SYNC_EXTERNAL:
 		if (!abort_is_user_exception(ai))
 			abort_print(ai);
-		DMSG("[abort] Ignoring async external abort!");
-		return FAULT_TYPE_IGNORE;
+		DMSG("[abort] %s!", fault_to_str(ai->abort_type,
```

Comment:
can remove the bang char `!`.
Can remove the leading space since since `fault_to_str()` prefixes a space char.